.github/travis/check_license.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
@acomodi - Can you create issues on each repository to track getting this enable this on all our other repositories?

Reply:
Sure